pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -166,6 +166,25 @@ func (c *HAProxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.Percon
 				Name:          "mysqlx",
 			},
 		)
+		appc.LivenessProbe = &cr.Spec.HAProxy.LivenessProbes
+		appc.ReadinessProbe = &cr.Spec.HAProxy.ReadinessProbes
+		appc.ReadinessProbe.Exec = &corev1.ExecAction{
+			Command: []string{"/usr/local/bin/readiness-check.sh"},
+		}
+		appc.LivenessProbe.Exec = &corev1.ExecAction{
+			Command: []string{"/usr/local/bin/liveness-check.sh"},
+		}
+		probsEnvs := []corev1.EnvVar{
+			{
+				Name:  "LIVENESS_CHECK_TIMEOUT",
+				Value: fmt.Sprint(cr.Spec.HAProxy.LivenessProbes.TimeoutSeconds),
+			},
+			{
+				Name:  "READINESS_CHECK_TIMEOUT",
+				Value: fmt.Sprint(cr.Spec.HAProxy.ReadinessProbes.TimeoutSeconds),
+			},
+		}
+		appc.Env = append(appc.Env, probsEnvs...)
```

Comment:
we should add them only if version >= 1.10

Reply:
I want to jump to the outgoing train but 1.10 is ok for me too.

Reply:
ok, lets do for 1.9

Reply:
ok